docs/source/executing-operations/defer-support.mdx

```diff
@@ -40,6 +40,19 @@ To learn the basics of the `@defer` directive and how you can use it with your s
 
 The remainder of this article covers the Apollo Router's defer implementation in greater depth.
 
+## Executing a `@defer` query
+
+To execute a deferred query on the Apollo Router, a GraphQL client sends an HTTP request with _almost_ the exact same format that it uses for usual query and mutation requests.
+
+The only difference is that the request should include the following `Accept` header:
```

Comment:
_Should_ we make this a "must"?
```suggestion
The only difference is that the request must include the following `Accept` header:
```

Reply:
Yes, I agree. If you don't send the header you won't get back a multipart response. Which essentially is not executing the defer query.